bravado/mapping/http_future.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+#
+# Copyright (c) 2013, Digium, Inc.
+# Copyright (c) 2014, Yelp, Inc.
+#
+
+DEFAULT_TIMEOUT_S = 5.0
+
+
+class HttpFuture(object):
```

Comment:
I think this needs to be moved out of `bravado/mapping` though into just `bravado`. This would never be used by the server.